platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulersConfig.java

```diff
@@ -102,4 +108,8 @@ public record PlatformSchedulersConfig(
         @ConfigProperty(defaultValue = "SEQUENTIAL") TaskSchedulerType stateSignatureCollectorSchedulerType,
         @ConfigProperty(defaultValue = "500") int stateSignatureCollectorUnhandledCapacity,
         @ConfigProperty(defaultValue = "SEQUENTIAL") TaskSchedulerType shadowgraphSchedulerType,
-        @ConfigProperty(defaultValue = "500") int shadowgraphUnhandledCapacity) {}
+        @ConfigProperty(defaultValue = "500") int shadowgraphUnhandledCapacity,
+        @ConfigProperty(defaultValue = "SEQUENTIAL_THREAD") TaskSchedulerType consensusRoundHandlerSchedulerType,
+        @ConfigProperty(defaultValue = "5") int consensusRoundHandlerUnhandledCapacity,
+        @ConfigProperty(defaultValue = "SEQUENTIAL") TaskSchedulerType eventStreamManagerSchedulerType,
```

Comment:
event stream manager already has a queue and a thread, this should probably be direct?

Reply:
The problem is that the component has multiple input wires, so it isn't possible to do direct scheduling

Reply:
You can lie and call it `DIRECT_STATELESS`, but only if it is safe for multiple threads to call into it simultaneously.

I've been thinking we should probably rename `DIRECT_STATELESS` into `DIRECT_THREAD_SAFE` or something similar. The important thing for this kind of scheduler is to be thread safe, and something that is stateless is trivially thread safe. But it isn't the only way you can be thread safe.

Reply:
@cody-littley Hmm, it looks to me like the streams used in `EventStreamManager` actually *aren't* threadsafe. There wasn't any protection before, but I suspect this might be thread unsafe code that happens to not conflict, based on when the various methods are called.

We could go the `DIRECT_STATELESS` route if we want to keep trusting the old behavior patterns, or leave this as a `SEQUENTIAL` scheduler (potentially with a much smaller capacity). I don't think touching the multistream code to make it threadsafe is a route we would ever want to take

Reply:
If there is a thread safety concern, better to stay with `SEQUENTIAL`, in my opinion.

Reply:
Based on recent issues and Kelly's message, we should try not to change the threading model if possible, I will take a look at the event stream manager and see what the conflicts could be

Reply:
> I will take a look at the event stream manager
 
I regret this now

Reply:
Even sequential would not be thread safe, since this thing has internal threads

Reply:
Based on the discussion, we will add simple thread safety to this class